File: isPrime.java
import java.util.*;
public class Main{
  public static void main(String args[]){
  
     System.out.println("Enter a Number:"); 
     Scanner sc = new Scanner(System.in);
     int n = sc.nextInt();
     if(isPrimeNumber(n)==true)
      System.out.println(n+" is a Prime Number");
     else
      System.out.println(n+" is  not a Prime Number");
 }
  
static boolean isPrimeNumber(int n) {
    if (n<2)
	return false;	    
    if (n==2)
	return true;
    for(int i=3;i*i<=n;i+=2) {
        if(n%i==0)
            return false;
    }
    return true;
}
}

File: swift/algorithms/IsPrime.swift
import Foundation

var flag: Bool = false;

let number : Int = 13;
for i in 2…number/2 {
    if(number % i == 0){
        flag = true
        break;
    }
}
if flag == false {
    print(“\(number ) is prime!”);
} else {
    print(“\(number ) is not prime!”);
}
